substrate/frame/staking-async/guide/src/overview.md

```diff
@@ -0,0 +1,174 @@
+small part of it on the Polkadot relay-chain (**RC** for short).
+
+The end goal for this system is for the staking system on AssetHub to elect a secure validator set for the relay-chain,
+and communicate it to the relay-chain in a timely fashion.
+
+
+The workings of the staking system can broadly be divided into the following components:
+
+```mermaid
+flowchart LR
+  subgraph RelayChain
+    StakingAhClient
+    Session
+    Historical
+    Timestamp
+    Authorship
+    Offences
+
+  end
+
+  subgraph AssetHub
+    subgraph Staking
+      AHSession["Session and Era Tracking"]
+      Exposures
+      Slashing
+      Rewards
+      User["User Ledgers"]
+
+      Slashing --> Exposures
+      Rewards --> Exposures
+    end
+
+    StakingRcClient
+
+    subgraph Election["Multi-Block Election Provider"]
+      Signed
+      Unsigned
+      Verifier
+    end
+
+    subgraph ElectionAlgorithms
+      SpNpos["sp-npos-elections"]
+    end
+
+    subgraph BagsList
+      VoterList
+    end
+
+    Election --ElectionProvider--> Staking
+    Staking --ElectionDataProvider--> Election
+    VoterList --> Staking
+    Election --> ElectionAlgorithms
+  end
+
+  StakingRcClient -- ValidatorSet --> StakingAhClient
+  StakingAhClient -- SessionReport+Offence --> StakingRcClient
+```
+
+The following sections describe each of these components in short, with a link to a dedicated chapter.
+
+## AssetHub Components
+
+Let's first describe the components that live on the AssetHub parachain.
+
+### User Ledgers/Interactions
+
+This is the part of the system that allows users to `bond` tokens, `unbond` them, and declare their intentions to
+`nominate`, `validate`, and so on.
+
+This system is fairly simple, not changed by much since the launch of the first iteration of Polkadot and its original
+`pallet-staking`, and is described in detail in the [User Interface](./user-interactions.md) chapter.
+
+### Sessions and Eras
+
+An era is the basic "_unit of time_" for the staking system. It is denominated in a number of smaller units called
+"sessions". The role of this subsystem is to keep track of the session/era progress, keep track of all of the associated
+data, and trigger the election in due time.
+
+More in [Sessions and Eras](./sessions-era.md).
+
+### Election and Exposures
+
+Following the above section, when the right time comes, the staking system will ask its `type ElectionProvider` to
+perform a new validator election and return the results to it. The process eventually completes, and is described in
+detail in the [Election and Exposures](./election.md) chapter.
+
+```rust,noplayground
+{{#include ../../src/pallet/mod.rs:election_provider}}
+```
+
+
+> To perform the election, both: `sp-npos-elections` and `VoterList` are used under the hood, which are not covered in
+> detail here
+
+Worth noting is that the final outcome of the election process is what we call `Exposure`: a `struct` identifying:
```

Comment:
We no longer use this unbounded version. It was previously used in one part of the slashing logic, but with staking-async, that usage is gone too. We should clean it up from sp-staking.

The currently used struct is `ExposurePage`, which works together with `PagedExposureMetadata`.